ui/imports/utils/Utils.qml

```diff
@@ -980,4 +980,24 @@ QtObject {
 
         return typeName
     }
+
+    function weiToEth(value) {
```

Comment:
That's nice to have those UI-side method there. I think next we could finally cleanup all those conversions because there are various stores doing that conversion in qml, incorrectly, e.g.:
```
    function getWei2Eth(wei, decimals) {
        return wei/(10**decimals)
    }
```
in `TransactionStore.qml`

Or e.g.:
```
    function getWei2Eth(wei) {
        return globalUtilsInst.wei2Eth(wei,18)
    }
 ```
in `RootStore.qml`, `SwapStore` and others